inputs/extracts_prompt_prefix.txt

```diff
@@ -0,0 +1,23 @@
+Suggest descriptions of images to illustrate for the following paragraphs from short stories:
```

Comment:
Wouldn't you only want to have this line here? The other text is story-specific that should be extracted automatically from a txt file.